lib/rack/tracker/criteo/criteo.rb

```diff
@@ -0,0 +1,31 @@
+class Criteo <  Rack::Tracker::Handler
+  class Event < OpenStruct
+    def write
+      to_h.to_json
+    end
+  end
+
+  self.position = :body
+
+  def tracker_options
```

Comment:
I would be -1 for users settings this manually for each criteo request.  It should be set once in an initializer.  But maybe I didn't fully understand the suggestion.